.github/workflows/tests.yml

```diff
@@ -37,3 +37,15 @@ jobs:
 
       - name: Run tests
         run: pytest
+
+  docker-build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: false
```

Comment:
So we don't deploy - it wouldn't work anyway since we didn't configure the secrets :)

Reply:
I didn't get that.

Reply:
This was more for ensuring that the `Dockerfile` is correct on CI so that it's built on branches, but not pushed - this that's handled by the release pipeline.

Reply:
By the way which secrets are you referring to? The Docker Hub deplyment is working fine?

Reply:
Exactly that - but they are not exposed to the action so it wont be able to access it.